### PR TITLE
Add collapse option for view card details

### PR DESCRIPTION
**Pull request by matiasgaratortiz**

### CODE REVIEW

File: src/app/js/controllers/cardController.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+"use strict";
```
Comment (leolower) [inline src/app/js/controllers/cardController.js]: :)

File: src/views/card-item.html [hunk 2]
```diff
@@ -7,9 +7,13 @@
     {{ card.votes.negatives}} <i class="glyphicon glyphicon-thumbs-down"></i>
   </span>
 
-  {{ card.title }} <small ng-show="card.completed"><strong>Completed!</strong></small>
+  <a href="" ng-click="isCollapsed = !isCollapsed">{{ card.title }} {{show_card[card.id]}}</a> <small ng-show="card.completed"><strong>Completed!</strong></small>
```
Comment (leolower) [inline src/views/card-item.html]: What does ` {{show_card[card.id]}}` do here?

File: src/index.html [hunk 3]
```diff
@@ -46,11 +48,14 @@
 
   
   <script src="vendor/angular/angular.min.js"></script>
+  <script src="vendor/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+  <script src="vendor/angular-ui-router/release/angular-ui-router.min.js"></script>
```
Comment (leolower) [inline src/index.html]: Do you need this?

**Comment by unjapones** [comment]
I'm assigning this back to you @matiasgaratortiz . The only things that I've found are the same @leolower already commented:

1) Removing the ui-router JS reference file from `index.html`.
2) Removing the `{{show_card[card.id]}}` from `card-item.html` since it's referencing something that doesn't exist in the source of the project (or may be add what is missing).

After finishing the items above, feel free to assign it back to me or merge it there aren't major changes

**Comment by matiasgaratortiz** [comment]
ui-router JS References is needed for Login.

**Pull request merged**